06.bowling_object/frame.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_reader :shots, :frame_idx
+
+  def initialize(frame_idx)
+    @shots = []
+    @frame_idx = frame_idx
+  end
+
+  def add_shot(marks)
+    loop do
+      shot = Shot.new(marks.shift)
+      @shots << shot.score
```

Comment:
```suggestion
      @scores << shot.score
```

なのでは？
https://bootcamp.fjord.jp/pages/readable-variable-name

その一方で、以下の記事も参考になりそうです。
https://bootcamp.fjord.jp/pages/utilize-objects

Reply:
`@scores`に`Shot`オブジェクトの配列を持たせ、オブジェクトの形を保持するようにしました。